### PR TITLE
More README fixes

### PR DESCRIPTION
**Pull request by dokun1**
Some changes to errors found by Enrique.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,8 @@ grade.save { (id: Int?, grade: Grade?, error: RequestError?) in
 }
 ```
 
+**NB**: If you want to use `RequestError`, you'll need to import `KituraContracts` at the top of your swift file.
```
Comment (EnriqueL8) [inline README.md]: We don't need this anymore since we created a public `typealias` for `KituraContracts` but it hasn't been tagged yet

Comment (EnriqueL8) [reply README.md]: now has been tagged as `0.0.2`

Comment (ianpartridge) [reply README.md]: @dokun1 can you remove this section? Once that's done I think this is good to go.

**Comment by dokun1** [comment]
@ianpartridge @EnriqueL8 thanks you two - pending travis and your comments, let me know if there's anything else!

**Pull request merged**